git_machete/cmd.py

```diff
@@ -847,7 +845,7 @@ def print_new_line(new_status: bool) -> None:
         for b in itertools.dropwhile(lambda x: x != cb, self.managed_branches):
             u = self.up_branch.get(b)
 
-            needs_slide_out: bool = is_merged_to_upstream(self, self.cli_ctxt, b)
+            needs_slide_out: bool = is_merged_to_upstream(self, b)
```

Comment:
In the spirit of my previous longish comment, I think `is_merged_to_upstream` can also be a part of `MacheteContext`

Reply:
This is a very machete-specific thing, and can be used programmatically as-is (no parsing of command-line input involved)